package.json

```diff
@@ -78,6 +78,7 @@
   },
   "dependencies": {
     "attr-accept": "^1.1.3",
+    "html5-file-selector": "^2.1.0",
```

Comment:
This needs to go into `devDependencies` since it should not become part of the distribution package.